scalableminds/webknossos
# Use s3 auth environment variables if set

Reads s3 credentials from environment variables if available (whenever none are supplied via UI)

### Steps to test:
- Explore remote dataset that needs S3 credentials via the Add Remote Dataset Tab, but do not supply credentials in the UI. Should fail
- Now restart webknossos with `AWS_ACCESS_KEY_ID=<value> AWS_SECRET_KEY=<value> yarn start`
- The same should work now

### Issues:
- fixes #7147

------
